routes/status.js

```diff
@@ -6,4 +7,14 @@ module.exports = (app) => {
   app.get('/status', (req, res) => {
     res.sendStatus(200);
   });
+
+  app.get('/message', (req, res) => {
+    MessageBroker.publish("Hello from Digital Paper Edit API", (err, data) => {
```

Comment:
Would be great if the broker method returned a promise too. I imagine the rest of the API will work that way later 👍

Reply:
Won't be doing this as part of https://github.com/bbc/digital-paper-edit-api/issues/2
Started trying some logic for Promises, and thought best to wait until async and await / refactor ticket coming along. This branch can be used as a starting point as it will remain open.